Source/control.h

```diff
@@ -56,6 +52,9 @@ extern bool panelflag;
 extern int initialDropGoldValue;
 extern bool panbtndown;
 extern bool spselflag;
+extern Rectangle rectMainPanel;
+extern Rectangle rectLeftPanel;
+extern Rectangle rectRightPanel;

```

Comment:
Don't prefix them with there type, and globals start with upper case:

This would work:
```suggestion
extern Rectangle MainPanel;
extern Rectangle LeftPanel;
extern Rectangle RightPanel;
```

Reply:
I will modify related part and update later.

Reply:
Modified in the last commit.